app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp, file, file_copy = command.split()
+    if cp != "cp":
+        return
+    if file == file_copy:
+        return
```

Comment:
```suggestion
    if cp == "cp" and file != file_copy:
        # do the copying
```
Так воно усе займе тільки один рядок, замість чотирьох